lib/contentful_delivery/spaces.ex

```diff
@@ -0,0 +1,65 @@
+defmodule Contentful.Delivery.Spaces do
+  @moduledoc """
+  Spaces provides function related to the reading of spaces
+  through the Contentful Delivery API
+  """
+
+  alias Contentful.{Delivery, MetaData, Space}
+
+  @doc """
+  will attempt to retrieve one space by it's space id
+
+  ## Examples for retrieving by Space ID
+
+      # space you have access to, token will be read from config/config.exs
+      iex> {:ok, space} = Contentful.Delivery.Spaces.fetch_one("space_id")
+      {:ok, %Contentful.Space{name: "a space name", meta_data: %{id: "space_id"}}}
+
+      # space that does not exist
+      iex> {:error, :not_found, original_message: _message}
+        = Contentful.Delivery.Spaces.fetch_one("non_existing_space", "<your_api_key>")
+
+      # no access
+      iex> {:error, :unauthorized, original_message: _message}
+        = Contentful.Delivery.Spaces.fetch_one("non_existing_space", "<your_api_key>")
+
+  In case you configured the space in `config/config.exs`:
+
+  ## Examples for plain call
+
+      # in config/config.exs
+      config :contentful, delivery: [
+        api_key: "<YOUR CDA_TOKEN>"
+        space: "myspace"
```

Comment:
Mentioning it here (but basically applies everywhere config related:
We should really stick to `space_id` as long as we also mean only the ID. Since a space struct does already exists, there were moments it confused me. I left a comment about it above in Assets.

Reply:
I'll try to standardize around `space_id`.